tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -243,23 +245,39 @@ Object.Pipeline {
 	}
 
 	# capture pipelines
-	passthrough-capture.1 {
+	gain-capture.1 {
 		index 	7
 		Object.Widget.pipeline.1 {
 			stream_name 'NoCodec-0'
 		}
 		Object.Widget.copier.1 {
 			stream_name 'SSP0 Capture'
 		}
+		Object.Widget.gain.1 {
+			Object.Control.mixer.1 {
+				name 'Main Capture Volume 1'
+			}
+		}
 	}
 
-	passthrough-be.4 {
-		index		8
-		direction	capture
+	gain-capture.4 {
+		index 	17
+		format	s32le
 		Object.Widget.pipeline.1 {
```

Comment:
remove gain for capture. it is not supported by test fw.  please check https://github.com/thesofproject/sof/issues/6367